lib/chinese_menu.rb

```diff
@@ -0,0 +1,32 @@
+class ChineseMenu
```

Comment:
Would expect to see a 'Menu' class, and be able to instantiate new instances of that class called 'chinese_menu', 'italian_menu'  etc, and these instances would have the specific menu information in them. Classes should be re-usable in different contexts.